src/components/Button.tsx

```diff
@@ -315,20 +314,14 @@ const IconButtonTemplate = styled(ButtonTemplate)<{
 export const IconButton = styled(IconButtonTemplate)`
   ${(props) =>
     btnColors(
-      props.theme.colors.text.primary,
+      '#6e6e6e',
       props.theme.colors.button.default.background.default,
       props.theme.colors.button.default.border.default,
       false
     )}
 
   &:not([disabled]):hover, &:not([disabled]):focus {
-    ${(props) =>
-      btnColors(
-        props.theme.colors.brand.medium,
-        'transparent',
-        'transparent',
-        false
-      )}
+    ${(props) => btnColors('#363636', '#f2f2f2', '#f2f2f2', false)}
```

Comment:
we use this button for multiple components that change will have a side effect on them, can we redeclare that button in `ContextMenu.tsx`

Reply:
OK... this change is reverted and button is styled in ContextMenu